app/models/conversion_host.rb

```diff
@@ -6,6 +6,7 @@ class ConversionHost < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   has_many :service_template_transformation_plan_tasks, :dependent => :nullify
   has_many :active_tasks, -> { where(:state => 'active') }, :class_name => ServiceTemplateTransformationPlanTask, :inverse_of => :conversion_host
+  delegate :ext_management_system, :to => :resource
```

Comment:
You might also want `:allow_nil => true` so that if the conversion host isn't linked to any inventory item this doesn't raise a NoMethodError but just returns nil